src/sage/calculus/calculus.py

```diff
@@ -349,6 +349,10 @@
 Note that ``x`` is still ``x``, since the
 maxima used by the calculus package is different than the one in
 the interactive interpreter.
+Clear the maxima variables to avoid interference with other tests::
+
+    sage: maxima('kill(x,y)')
+    done
```

Comment:
what exactly is the failure? Since none of the code below appears to use something maxima-specific it feels like if the test result depends on some maxima variable it would be a bug (?)

Reply:
There are a few other doctests that explicit call `maxima` using the same variables `x,y` but require them to be unspecified.